free-programming-books-et.md

```diff
@@ -30,15 +33,29 @@
 ### JavaScript
 
 * [JavaScript](http://puhang.tpt.edu.ee/raamatud/JavaScript_konspekt.pdf) - Jüri Puhang (PDF)
+* [Javascript](http://lepo.it.da.ut.ee/~epuman/javascript.html) - Ella Puman, Tartu ÜLikooli õppematerjal
 
 
+##### Angular
+
+* [Angular](http://www.cs.tlu.ee/teemad/get_file.php?id=400) - Tallinna Ülikooli õppematerjal
```

Comment:
link didn't work

Reply:
![image](https://user-images.githubusercontent.com/33719534/195769324-a31dbe4c-7e40-4490-a8b6-781b26b1e99d.png)
Seems to work for me. Download starts automatically.